### PR TITLE
Get PropTypes from prop-types package

### PR DESCRIPTION
**Pull request by felixhorro**
React.PropTypes is deprecated as of React v15.5. Changing to read PropTypes from prop-types package, as [seen on React docs](https://facebook.github.io/react/docs/typechecking-with-proptypes.html).

### CODE REVIEW

**Comment by Guria** [comment]
Would be great to see here extending peer deps to support "react@^16.0.0-0" as well.

**Comment by taion** [comment]
That would be a separate PR.

File: package.json [hunk 1]
```diff
@@ -63,6 +63,7 @@
     "eslint-plugin-jsx-a11y": "^4.0.0",
     "eslint-plugin-react": "^6.10.0",
     "jest-cli": "^19.0.2",
+    "prop-types": "^15.5.6",
```
Comment (taion) [inline package.json]: this needs to be a normal dependency

Comment (jquense) [inline package.json]: GOOD CATCH

Comment (felixhorro) [inline package.json]: Ok, nice point!

Will commit.

**Comment by liady** [comment]
@taion any news on this PR?

**Pull request merged**